### ISSUE TITLE
Is Phonon now dead?

### ISSUE THREAD

**Issue by UnyteLux** [quark-dev/Phonon-Framework#254]
I notice that there have been no code updates in 91 days now and most recent issues opened here have gone unanswered.  Does this mean that Phonon has now been abandoned by its developers?

**Comment by John-Henrique** [comment]
No, isn't. This mean developers are busy working to get money or simple no have time to do things here at this moment.

In any case, all users can help Phonon project...
